tcms/core/tests/test_report_tag.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+import unittest
+from tcms.core.templatetags.report_tags import percentage
+
+
+class TestReportTagPercentage(unittest.TestCase):
+
+    def test_percentage_calculation(self):
+
+        result = percentage(3.0, 2.0)
+        self.assertTrue(isinstance(result, str))
+        self.assertEqual(percentage(5.0, 2.5), '200.0%')
```

Comment:
The match to a string constant explicitly checks the type as well so you don't need the above 2 lines before this assert.

Reply:
What's important in the result of the `percent()` function is that it always returns 1 character after the decimal divider. 

so a 1 % becomes "1.0%",
1.2 -> "1.2%"
and 1.23 -> "1.2%" again

You need test scenarios for all of these.